src/core/EspressoSystemInterface_cuda.cu

```diff
@@ -84,13 +84,21 @@ __global__ void split_kernel_v(CUDA_particle_data *particles, float *v, int n) {
   if (idx >= n)
     return;
 
+#ifdef LB_GPU
```

Comment:
This is redundant, the whole function is `#ifdef LB_GPU`, the `elif` branch is never taken.